ccw.core/src/clj/ccw/repl/view_helpers.clj

```diff
@@ -1,37 +1,32 @@
 (ns ccw.repl.view-helpers
   (:require [clojure.tools.nrepl :as repl]
-    [ccw.repl.cmdhistory :as history]
-    [ccw.events :as evt])
-  (:use [clojure.core.incubator :only (-?>)]
-        [clojure.tools.nrepl.misc :only (uuid)])
+            [clojure.tools.nrepl.misc :refer (uuid)]
+            [ccw.repl.cmdhistory :as history]
+            [ccw.events :as evt]
+            [ccw.eclipse :as eclipse])
   (:import ccw.CCWPlugin
-    org.eclipse.ui.PlatformUI
-    org.eclipse.swt.SWT
-    (org.eclipse.swt.custom StyledText StyleRange))
-  (:import
-    [org.eclipse.ui.handlers HandlerUtil]))
+           org.eclipse.ui.PlatformUI
+           org.eclipse.swt.SWT
+           [org.eclipse.swt.custom StyledText StyleRange]
+           org.eclipse.ui.handlers.HandlerUtil
+           ccw.util.DisplayUtil))
 
-(defn workbench-display
-  []
-  (-> (PlatformUI/getWorkbench) .getDisplay))
+(def workbench-display eclipse/workbench-display)
 
-(defmacro ui-async
```

Comment:
On my laptop, ansi-repl fails to load because it was referencing e/async-ui but not display-handlers/async-ui. Will fix this locally (I have created a new PR with new branch and just he async stuff).